e2e/tests/config/testdata/profile-activation/regexp-exact.yaml

```diff
@@ -0,0 +1,18 @@
+version: v1beta10
```

Comment:
Shouldn't this be v1beta11?

Reply:
Yeah would be good to use the newest config version here